lxc/exec_unix.go

```diff
@@ -69,74 +71,85 @@ func (c *cmdExec) controlSocketHandler(control *websocket.Conn) {
 				err = c.forwardSignal(control, unix.SIGTERM)
 				sig = unix.SIGTERM
 			}
+
 			logger.Debugf("Received '%s signal', forwarding to executing program.", sig)
 			if err != nil {
 				logger.Debugf("Failed to forward signal '%s'.", sig)
 				return
 			}
+
 		case unix.SIGINT:
 			logger.Debugf("Received '%s signal', forwarding to executing program.", sig)
 			err := c.forwardSignal(control, unix.SIGINT)
 			if err != nil {
 				logger.Debugf("Failed to forward signal '%s'.", unix.SIGINT)
 				return
 			}
+
 		case unix.SIGQUIT:
 			logger.Debugf("Received '%s signal', forwarding to executing program.", sig)
 			err := c.forwardSignal(control, unix.SIGQUIT)
 			if err != nil {
 				logger.Debugf("Failed to forward signal '%s'.", unix.SIGQUIT)
 				return
 			}
+
 		case unix.SIGABRT:
 			logger.Debugf("Received '%s signal', forwarding to executing program.", sig)
 			err := c.forwardSignal(control, unix.SIGABRT)
 			if err != nil {
 				logger.Debugf("Failed to forward signal '%s'.", unix.SIGABRT)
 				return
 			}
+
 		case unix.SIGTSTP:
 			logger.Debugf("Received '%s signal', forwarding to executing program.", sig)
 			err := c.forwardSignal(control, unix.SIGTSTP)
 			if err != nil {
 				logger.Debugf("Failed to forward signal '%s'.", unix.SIGTSTP)
 				return
 			}
+
 		case unix.SIGTTIN:
 			logger.Debugf("Received '%s signal', forwarding to executing program.", sig)
 			err := c.forwardSignal(control, unix.SIGTTIN)
 			if err != nil {
 				logger.Debugf("Failed to forward signal '%s'.", unix.SIGTTIN)
 				return
 			}
+
 		case unix.SIGTTOU:
 			logger.Debugf("Received '%s signal', forwarding to executing program.", sig)
 			err := c.forwardSignal(control, unix.SIGTTOU)
 			if err != nil {
 				logger.Debugf("Failed to forward signal '%s'.", unix.SIGTTOU)
 				return
 			}
+
 		case unix.SIGUSR1:
 			logger.Debugf("Received '%s signal', forwarding to executing program.", sig)
 			err := c.forwardSignal(control, unix.SIGUSR1)
 			if err != nil {
 				logger.Debugf("Failed to forward signal '%s'.", unix.SIGUSR1)
 				return
 			}
+
 		case unix.SIGUSR2:
 			logger.Debugf("Received '%s signal', forwarding to executing program.", sig)
 			err := c.forwardSignal(control, unix.SIGUSR2)
 			if err != nil {
 				logger.Debugf("Failed to forward signal '%s'.", unix.SIGUSR2)
 				return
 			}
+
```

Comment:
This check seems to be going a little too far in my view, as the `case` statements are still part of the overall block. 
What do you think? Should we make an exception here?

Reply:
I agree that it's taking it a little far. When I initially wrote this check for LXD Cloud I wasn't able to make this exception (I simply found it too difficult, I am sure sed is capable).